packages/material-ui/src/Box/Box.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { assert } from 'chai';
+import { createShallow } from '@material-ui/core/test-utils';
+import Box from './Box';
+
+describe('<Box />', () => {
+  let shallow;
+
+  before(() => {
+    shallow = createShallow({ dive: true });
```

Comment:
Can we use the mount API? We are moving away from shallow.

Reply:
Sure, I will do it tomorrow.

Reply:
Thank you